modules/class.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
```

Comment:
- [ ] I understand that you are struggling with `eslint` errors here, I would like to suggest that you transfer some _**related classes**_ to a new file in the modules folder. That way, you can separate concerns and your code is easy to maintain. 👍